src/drivers/timer/src/hpet.rs

```diff
@@ -82,6 +83,16 @@ impl HPET {
         self.base as *const _
     }
 
+    pub fn enable(&self) -> Result<(), &str> {
+        // set enable bit for HPET timer as per 2.3.5
+        // https://www.intel.com/content/dam/www/public/us/en/documents/technical-specifications/software-developers-hpet-spec-1-0a.pdf
+        self.config.set(1); 
```

Comment:
Instead of doing a raw set, you can use `config.modify(CONFIG::TMREN::SET)` since Danny already defined the type on L63

Reply:
I have to take a closer look at this tock_registers crate. Looks great, thanks!

Reply:
Resolved. Danny showed me another trick, which is great.